src/Analysis/Ast/Test/ImportTests.cs

```diff
@@ -234,5 +238,20 @@ import sys
             var analysis = await GetAnalysisAsync(code);
             analysis.Should().HaveVariable("x").OfType(BuiltinTypeId.Int);
         }
+
+        [TestMethod, Priority(0)]
+        public async Task ModuleMembers() {
+            var appUri = TestData.GetTestSpecificUri("app.py");
+            await TestData.CreateTestSpecificFileAsync(Path.Combine("package", "__init__.py"), "from . import m1\nimport sys");
+            await TestData.CreateTestSpecificFileAsync(Path.Combine("package", "m1", "__init__.py"), string.Empty);
+
+            await CreateServicesAsync(PythonVersions.LatestAvailable3X);
+            var rdt = Services.GetService<IRunningDocumentTable>();
+            var doc = rdt.OpenDocument(appUri, "import package");
+
+            await Services.GetService<IPythonAnalyzer>().WaitForCompleteAnalysisAsync();
+            var analysis = await doc.GetAnalysisAsync(Timeout.Infinite);
+            analysis.Should().HaveVariable("package").Which.Should().HaveMember("m1").And.NotHaveMember("sys");
```

Comment:
This assertion is incorrect in both cases:
- `package.sys` is a real member of the package
- `m1` doesn't have to be submodule. It is actually doesn't matter where module is located. What matters is if the name is present in `sys.modules`, which effectively depends on loading order. Here is an example:
   - File structure
    ```
    ├─ package
    │  ├─ sub1
    │  │  └─ m1
    │  │     └─ __init__.py
    │  ├─ sub2
    │  │  └─ m2.py
    │  ├─ m3.py
    │  └─ __init__.py
    └─ app.py
    ```
  - `package/sub1/m1/__init__.py`:

    ```py
    import sys
    ```
  - `package/sub2/m2.py`:

    ```py
    from package.sub1 import m1 as p
    ```
  - `package/__init__.py`:

    ```py
    import package.sub2.m2
    ```
  - `app.py`:

    ```py
    from sys import modules
    import package
    print('package.sub1.m1' in modules, hasattr(package.sub1, 'm1'), sep=', ') # True, True
    print('package.sub1.m1.sys' in modules, hasattr(package.sub1.m1, 'sys'), sep=', ') # False, True
    print('package.sub1.m3' in modules, hasattr(package.sub1, 'm3'), sep=', ') # False, False
    ```

Reply:
Sys is indeed a member but we don’t want to show it since otherwise we will show everything module imports. We never showed it before. User can type it if they wish but that is not something we offer. Basically we relax the filter and allow imported sub modules as users want

Reply:
So the point is to resolve user request to see pandas.testing and torch.mm

Reply:
The problem is, as @jakebailey mentioned, that there can be no member, even if module itself exists (see the last line with `package.m3` in my example).

Reply:
In the filter we already know it is present in the module as a variable. I.e. `GetMemberNames` goes through existing variables so it won't pick up something that is not imported. The difference is that before we did not show any of the imported modules and now we allow those variables that hold imported submodules.

Reply:
Yeah, this is the middle ground we came up with for the most common case (re-exporting something in the project out).

Another solution entirely might be to just let _everything_ be a member, then start filtering at completion time. My worry with not adding things as members is that it's valid code to write something like `from package.sub1.m1 import sys`, which might be important to preserve. If our issue with re-exporting every member is that it's annoying for users, maybe deferring the filter is the right way to go. (Sorry to suggest yet another solution, just brainstorming)

Reply:
Maybe worth noting the rules on when imports are considering exporting a type stub (from https://www.python.org/dev/peps/pep-0484/):

```
Additional notes on stub files:

Modules and variables imported into the stub are not considered exported from the stub unless the import uses the import ... as ... form or the equivalent from ... import ... as ... form.

However, as an exception to the previous bullet, all objects imported into a stub using from ... import * are considered exported. (This makes it easier to re-export all objects from a given module that may vary by Python version.)
```

Reply:
Okay, that's definitely useful knowledge as I don't think we treat stubs any differently when handling `*` imports.

Reply:
Yes, I thought about filtering at completion, which is more appropriate, but it requires creating something more complex than GetMemberNames, like `GetMemberInfos` which would also provide additional data on what the member is exactly.

Reply:
Should be resolved in the latest update